gradle/java.gradle

```diff
@@ -21,8 +21,8 @@ compileJava {
             "-Xlint:fallthrough",
             "-Xlint:finally",
             "-Xlint:overrides",
-            "-Xlint:path",
-            "-Werror"
+            "-Xlint:path"
+//            "-Werror"
```

Comment:
BTW, Why was this commented out?

Reply:
That was a mistake, the compatibility `.VERSION_1_8` in the `build.gradle` file is deleted by mistake. I will fix it.

Reply:
The gradle will treat deprecation as error with `-Werror` turned on. In the current code base, Flink deprecated `Types`, `BatchTableSink` and `BatchTableSource` are imported and used. Recent commit fixed the compatability and all the `Types`. To fully fix `BatchTableSink` and `BatchTableSource`, we might reconsider the batch table api design. I think we can leave it here to treat it as a warning.